src/interfaces/IMorpho.sol

```diff
@@ -139,6 +144,8 @@ interface IMorpho {
     /// - The token balance of the sender (resp. receiver) should decrease (resp. increase) by exactly the given amount
     /// on `transfer` and `transferFrom`. In particular, tokens with fees on transfer are not supported.
     /// - The IRM should not re-enter Morpho.
+    /// - The oracle should return a price with the correct scaling.
+    /// - The oracle is trusted to handle stale prices and price manipulations if necessary.
```

Comment:
Is this last assumption correct actually? I don't think we make such assumption on the oracle

Reply:
The comment above is the following:

```solidity
    /// @dev Here is the list of assumptions on the market's dependencies (tokens, IRM and oracle) that guarantees
    /// Morpho behaves as expected:
```

How to we say that the oracle must behave well as well?

Reply:
>Is this last assumption correct actually? I don't think we make such assumption on the oracle

yes, this is not needed in order for Morpho to behave as expected

Reply:
Just to be clear, I'm only talking about the line:
`The oracle is trusted to handle stale prices and price manipulations if necessary.`

To me, the assumption:
`The oracle should return a price with the correct scaling.` is required to have Morpho behave as expected, but not the other one (whether price are manipulated or stale are not considered by Morpho: the protocol will just consume the price of the oracle the market is attached to)

Reply:
But I think the following should be written somewhere:

> the protocol will just consume the price of the oracle the market is attached to, without any assumption on it

Reply:
Perhaps should we move that comment to the struct? "The oracle is trusted to handle stale prices and price manipulations if necessary."